contracts/proxy-lib/tests/config_helper.rs

```diff
@@ -0,0 +1,104 @@
+#[path = "test_utils.rs"]
+mod test_utils;
+
+use calimero_context_config::repr::{Repr, ReprTransmute};
+use calimero_context_config::types::{Application, ContextId, ContextIdentity, Signed, SignerId};
+use calimero_context_config::{ContextRequest, ContextRequestKind, Request, RequestKind};
+use ed25519_dalek::{Signer, SigningKey};
+use eyre::Result;
+use near_workspaces::{network::Sandbox, result::ExecutionFinalResult, Account, Contract, Worker};
+use rand::Rng;
+use test_utils::deploy_contract;
+
+const CONTEXT_CONFIG_WASM: &str = "../context-config/res/calimero_context_config_near.wasm";
+
+#[derive(Clone)]
+pub struct ConfigContractHelper {
+    pub config_contract: Contract,
+}
+
+impl ConfigContractHelper {
+    pub async fn new(worker: &Worker<Sandbox>) -> Result<Self> {
+        let config_contract = deploy_contract(worker, CONTEXT_CONFIG_WASM).await?;
+        Ok(Self {
+            config_contract,
+        })
+    }
+
+    pub async fn add_context_to_config(
+        &self,
+        caller: &Account,
+        context: &SigningKey,
+        author: &SigningKey,
+    ) -> Result<ExecutionFinalResult> {
+        let mut rng = rand::thread_rng();
+
+        let application_id = rng.gen::<[_; 32]>().rt()?;
+        let blob_id = rng.gen::<[_; 32]>().rt()?;
+
+        let author_id: Repr<ContextIdentity> = Repr::new(author.verifying_key().to_bytes().rt()?);
+        let context_id: Repr<ContextId> = Repr::new(context.verifying_key().to_bytes().rt()?);
+        let context_signer: Repr<SignerId> = Repr::new(context.verifying_key().to_bytes().rt()?);
+
+        let signed_request = Signed::new(
+            &{
+                let kind = RequestKind::Context(ContextRequest::new(
+                    context_id,
+                    ContextRequestKind::Add {
+                        author_id,
+                        application: Application::new(
+                            application_id,
+                            blob_id,
+                            0,
+                            Default::default(),
+                            Default::default(),
```

Comment:
More idiomatic to use the specific type names here, instead of `Default`.

Reply:
I'm afraid this is coming from the config-contract, I'll create an issue to address it in a different commit. Leaving this open until the issue is created